paper/paper.md

```diff
@@ -0,0 +1,190 @@
+---
+title: 'Pyglmnet: Python implementation of elastic-net regularized generalized linear models'
+tags:
+  - Python
+  - glm
+  - machine-learning
+  - lasso
+  - elastic-net
+  - group-lasso
+authors:
+  - name: Mainak Jas
+    orcid: 0000-0002-3199-9027
+    affiliation: "1, 2"
+  - name: Titipat Achakulvisut
+    affiliation: 3
+  - name: Aid Idrizović
+    affiliation: 4
+  - name: Daniel Acuna
+    affiliation: 5
+  - name: Matthew Antalek
+    affiliation: 6
+  - name: Vinicius Marques
+    affiliation: 4
+  - name: Tommy Odland
+    affiliation: 7
+  - name: Ravi Prakash Garg
+    affiliation: 6
+  - name: Mayank Agrawal
+    affiliation: 8
+  - name: Yu Umegaki
+    affiliation: 9
+  - name: Peter Foley
+    orcid: 0000-0002-0304-7213
+    affiliation: 10
+  - name: Hugo Fernandes
+    orcid: 0000-0002-0168-4104
+    affiliation: 11
+  - name: Drew Harris
+    affiliation: 12
+  - name: Beibin Li
+    affiliation: 13
+  - name: Olivier Pieters
+    orcid: 0000-0002-5473-4849
+    affiliation: 14, 20
+  - name: Scott Otterson
+    affiliation: 15
+  - name: Giovanni De Toni
+    orcid: 0000-0002-8387-9983
+    affiliation: 16
+  - name: Chris Rodgers
+    orcid: 0000-0003-1762-3450
+    affiliation: 17
+  - name: Eva Dyer
+    affiliation: 18
+  - name: Matti Hamalainen
+    affiliation: "1, 2"
+  - name: Konrad Kording
+    affiliation: 3
+  - name: Pavan Ramkumar
+    orcid: 0000-0001-7450-0727
+    affiliation: 19
+affiliations:
+ - name: Massachusetts General Hospital
+   index: 1
+ - name: Harvard Medical School
+   index: 2
+ - name: University of Pennsylvania
+   index: 3
+ - name: Loyola University
+   index: 4
+ - name: University of Syracuse
+   index: 5
+ - name: Northwestern University
+   index: 6
+ - name: Sonat Consulting
+   index: 7
+ - name: Princeton University
+   index: 8
+ - name: NTT DATA Mathematical Systems Inc
+   index: 9
+ - name: 605
+   index: 10
+ - name: Rockets of Awesome
+   index: 11
+ - name: Epoch Capital
+   index: 12
+ - name: University of Washington
+   index: 13
+ - name: IDLab-AIRO -- Ghent University -- imec
+   index: 14
+ - name: Clean Power Research
+   index: 15
+ - name: University of Trento
+   index: 16
+ - name: Columbia University
+   index: 17
+ - name: Georgia Tech
+   index: 18
+ - name: System1 Biosciences Inc
+   index: 19
+ - name: Research Institute for Agriculture, Fisheries and Food
+   index: 20
+
+date: 6 September 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+[Generalized linear models](GLMs) are
+well-established tools for regression and classification and are widely
+applied across the sciences, economics, business, and finance. They are
+especially useful because their loss functions are convex, meaning they are
```

Comment:
Perhaps rewrite to "..because their loss functions are convex, which makes fitting the models easy and efficient." My reasoning being that: They as easy to fit as a result of being convex, but convexity does not mean "easy to fit," it means [definition of convex here].

Reply:
Suggestion: "Owing to their convex loss, they are easy and efficient to fit"

Reply:
done